LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-request-resources.https.html

```diff
@@ -90,7 +90,67 @@
       destination: 'script',
       message: `Script load (url:${actual_url})`
     };
-  frame.contentWindow.load_script_with_integrity(actual_url, integrity);
+  frame.contentWindow.load_script_with_integrity(actual_url, integrity, /*type=*/null);
```

Comment:
Passing null is a bit confusing here, because null has no meaning in the context of load_script_with_integrity(), which just does a falsy check. 

```suggestion
  frame.contentWindow.load_script_with_integrity(actual_url, integrity);
```

Reply:
Done